junit-newrelic-processor/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM alpine:3.10
+
+# install libs to process and push xml to newrelic
+RUN apk update && \
+    apk add xmlstarlet curl bash sudo
```

standard practice to keep image layers smaller
```suggestion
RUN apk add --no-cache --update xmlstarlet curl bash sudo
```